packages/cli/src/util/env/diff-env-files.ts

```diff
@@ -0,0 +1,77 @@
+import { Output } from '../output';
+import { Dictionary } from '@vercel/client';
+import { readFile } from 'fs-extra';
+import { parseEnv } from '../parse-env';
+import chalk from 'chalk';
+
+export async function createEnvObject(
+  envPath: string,
+  output: Output
+): Promise<Dictionary<string | undefined> | undefined> {
+  try {
+    // Partially taken from https://github.com/tswaters/env-file-parser/blob/master/lib/parse.js
+    let envArr = (await readFile(envPath, 'utf-8'))
+      // remove double quotes
+      .replace(/"/g, '')
+      // split on new line
+      .split(/\r?\n|\r/)
+      // filter comments
+      .filter(line => /^[^#]/.test(line))
+      // needs equal sign
+      .filter(line => /=/i.test(line));
+    return parseEnv(envArr);
+  } catch (err) {
+    output.debug(`Error parsing env file: ${err}`);
+  }
+}
+
+function findChanges(
+  oldEnv: Dictionary<string | undefined>,
+  newEnv: Dictionary<string | undefined>
+): {
+  added: string[];
+  changed: string[];
+  removed: string[];
+} {
+  let added = [];
+  let changed = [];
```

Comment:
never re-assigned so these can be `const`
```suggestion
  const added = [];
  const changed = [];
```